crates/bevy_app/src/main_schedule.rs

```diff
@@ -71,6 +71,9 @@ pub struct RunFixedUpdateLoop;
 ///
 /// The exclusive `run_fixed_update_schedule` system runs this schedule.
 /// This is run by the [`RunFixedUpdateLoop`] schedule.
+///
+/// Frequency of execution is configured by inserting `Time<Fixed>` resource, 64 Hz by default.
```

Comment:
Can we link to the time type with the generic? maybe as two links?

Reply:
`bevy_app` does not depend on `bevy_time`. We can only link to docs.rs I think?

Reply:
Hah, didn't expect that, but makes sense. I don't think we need to link to it then :)